packages/client/src/theme.jsx

```diff
@@ -0,0 +1,61 @@
+import { createTheme } from '@mui/material/styles';
+
+// Compose theme in several steps so we can
+//   (1) use augmentColor, and
+//   (2) style components using named colors.
+export let theme = createTheme({});
+
+theme = createTheme(theme, {
+  palette: {
+    // Since we have a tricolor palette, and MUI's default palette system is bicolor,
+    // let's use named custom colors instead of adding a custom 'tertiary' to the built-in
+    // 'primary'/'secondary'. This will avoid surprises down the line when changing props
+    // on MUI components (for example, a developer might try to change indicatorColor
+    // or textColor on MUI <Tabs> from 'secondary' to 'tertiary', which will not work).
+    purple: theme.palette.augmentColor({
+      color: {
+        main: '#A888C7',
+      },
+    }),
+    yellow: theme.palette.augmentColor({
+      color: {
+        main: '#FFD884',
+      },
+    }),
+    blue: theme.palette.augmentColor({
+      color: {
+        main: '#78CEE9',
+      },
+    }),
+    black: theme.palette.augmentColor({
+      color: {
+        main: '#000000',
+      },
+    }),
+  },
+});
+
+theme = createTheme(theme, {
```

Comment:
```suggestion
```

Reply:
Here I am following this hint https://mui.com/material-ui/customization/theming/#api 

> Theme composition: using theme options to define other options
> When the value for a theme option is dependent on another theme option, you should compose the theme in steps.

in order to use color names (and `.light` etc) instead of repeating the hex codes